givitsite/friendreq/tests.py

```diff
@@ -1,3 +1,47 @@
-from django.test import TestCase
+from django.test import TestCase 
+from django.shortcuts import reverse
+from django.contrib.auth.models import User
+from .models import ItemRequest,ItemsFound
+from django.conf import settings
 
-# Create your tests here.
+class FriendPageTests(TestCase):
+    # check the status code when navigate to request page
```

Comment:
Since the 4 tests below essnetially do the same thing just for different URLs - I recommend writing a loop over a list - it will make it easier to expand the test when you add more pages

maybe this can even be combined with th 5th if you make all the tests also check the template...

Reply:
fixed